pkgs/development/python-modules/calmjs/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, calmjs-types
+, calmjs-parse
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "calmjs";
+  version = "3.4.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-DKR5INwfjH2zGQc+eBGmo58ct73Ta11C/SIQlNOiIYU=";
+    extension = "zip";
+  };
+
+  propagatedBuildInputs = [
+    calmjs-parse
+    calmjs-types
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
```

Comment:
Can you try the following:

```
preCheck = ''
  rm -rf src/calmjs
'';
```

Reply:
After that pytestCheckHook ran 0 tests :(

Reply:
Then I am out of ideas.

Reply:
you could try to set PYTHONPATH to $out/${python.sitePackages} but we can also continue to try this later